module2/gradlew

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
```

Comment:
do we need this file here?

Reply:
I believe the file is needed into the project structure. Also, I had uploaded the .gitignore file from the official repository.